src/libDirectoryService/DSBlockPostProcessing.cpp

```diff
@@ -460,18 +446,30 @@ void DirectoryService::StartFirstTxEpoch() {
       }
     }
 
+    // If I cannot find myself in the sharding structure, it means I must have
+    // been a non-performant node and must rejoin as normal.
     if (!found) {
       LOG_EPOCH(WARNING, m_mediator.m_currentEpochNum,
-                "WARNING: Oldest DS node not in any of the new shards!");
+                "WARNING: My DS node is not in any of the new shards! I must "
```

Comment:
Please change to:
`LOG_EPOCH(WARNING, m_mediator.m_currentEpochNum, "My DS node signed insufficient blocks. Kicked out and invoking RejoinAsNormal now.");`

Reply:
Thanks for the clearer message suggestion. It has been changed.